### PR TITLE
J-181: added sorting genres options

### PR DESCRIPTION
**Pull request by JuliaDzyuba**

### CODE REVIEW

File: frontend/src/components/configurate-podcast/components/configurate-podcast-form/configurate-podcast-form.tsx [hunk 1]
```diff
@@ -39,6 +39,8 @@ const ConfiguratePodcastForm: React.FC<Props> = ({
 
   const isFormDisabled = createPodcastStatus === DataStatus.PENDING;
 
+  const sortedGenres = genres.sort((a, b) => a.label > b.label ? 1 : -1);
```
Comment (what1s1ove) [inline frontend/src/components/configurate-podcast/components/configurate-podcast-form/configurate-podcast-form.tsx]: sort mutates an array
firstly we need to copy this array

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: frontend/src/components/configurate-podcast/components/configurate-podcast-form/configurate-podcast-form.tsx (hunk 1 above)
Comment (yevhenprybytok) [inline frontend/src/components/configurate-podcast/components/configurate-podcast-form/configurate-podcast-form.tsx]: Do we sort it by the length of the word?

Comment (Denidzio) [inline frontend/src/components/configurate-podcast/components/configurate-podcast-form/configurate-podcast-form.tsx:42]: Does `label` field has string type? JavaScript `.sort` sorts strings by default and we can remove `(a, b) => a.label > b.label ? 1 : -1` .

File: frontend/src/components/configurate-podcast/components/configurate-podcast-form/configurate-podcast-form.tsx [hunk 2]
```diff
@@ -39,6 +39,8 @@ const ConfiguratePodcastForm: React.FC<Props> = ({
 
   const isFormDisabled = createPodcastStatus === DataStatus.PENDING;
 
+  const sortedGenres = [...genres].sort((a, b) => a.label > b.label ? 1 : -1);
```
Comment (what1s1ove) [inline frontend/src/components/configurate-podcast/components/configurate-podcast-form/configurate-podcast-form.tsx]: you need to create a function from it and move it to `frontend/src/components/configurate-podcast/components/configurate-podcast-form/helpers`

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: frontend/src/components/configurate-podcast/helpers/sort-genres-by-name/sort-genres-by-name.ts [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import { Option } from 'common/types/types';
+
+const sortGenresByName = (options: Option[]): Option[] => {
+  return [...options].sort((a, b) => a.label > b.label ? 1 : -1);
```
Comment (what1s1ove) [inline frontend/src/components/configurate-podcast/helpers/sort-genres-by-name/sort-genres-by-name.ts]:
```suggestion
  return [...options].sort((a, b) => a.label.localeCompare(b.label));
```

**Review by what1s1ove — CHANGES_REQUESTED** [review]

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**